common/src/Microsoft.Azure.IIoT.Abstractions/src/Exceptions/MethodCallStatusException.cs

```diff
@@ -39,7 +39,7 @@ public class MethodCallStatusException : MethodCallException {
         /// <param name="errorMessage"></param>
         public MethodCallStatusException(string responsePayload, int result,
             string errorMessage = null) :
-            base($"Response {result} {errorMessage ?? ""}: {responsePayload}") {
+            base($" {{ \"Message\":\"Response {result} {errorMessage ?? ""}\",\"Description\": {responsePayload} }}") {
```

Comment:
Please add tests that would validate that we can reserialize `ResponsePayload` as a proper JSON afterwards. The tests should also check the cases where passed `responsePayload` or `errorMessage` are null. In both cases we should still be able to deserialize. I think that the test should also make sure that message constructed from the other constructor above is also deserializable.

Reply:
Added unit test